src/SDKs/HDInsight/Management/Management.HDInsight.Tests/.gitignore

```diff
@@ -0,0 +1 @@
+realsettings.json

```

Comment:
Is this intended?

Reply:
Yes. some sensitive data will be put to this file. in case of checking in the file by mistake, I add it in the git ignore file.